docs/tutorial/using-checkbox/manifest.rst

```diff
@@ -0,0 +1,300 @@
+    Outcome: job passed
+
+and as a result, the job that depends on it is skipped:
+
+.. code-block:: none
+    :emphasize-lines: 5-6
+
+    ---------------------------[ A job using a manifest ]---------------------------
+    ID: com.canonical.certification::tutorial/manifest
+    Category: com.canonical.certification::tutorial
+    Job cannot be started because:
+     - resource expression "manifest.has_touchscreen == 'True'" evaluates to false
+    Outcome: job cannot be started
+
+Modifying the manifest information manually
+===========================================
+
+Let's check the content of the manifest file:
+
+.. code-block:: none
+
+    $ cat /var/tmp/checkbox-ng/machine-manifest.json
+	
+    {
+      "com.canonical.certification::has_touchscreen": false
+    }
+
+As you can see, it's a JSON file that stores the value for each of the entries
+(only one in our case :)) we need to run our test plan.
+
+Using your favorite text editor, open the file, replace ``false`` with
+``true`` and save your modifications.
+
+Run Checkbox again, following the same steps we have done so far, and you will
+see in the System Manifest screen that ``Touchscreen`` is now set to ``Yes``.
+
+It can be very useful to provide pre-filled manifest files, especially when
+you want to automate Checkbox testing.
+
+Skipping the System Manifest screen
+===================================
+
+So far, every time you started Checkbox to run this test plan, you were
+greeted with the System Manifest screen to confirm the manifest information
+was correct prior to starting the test.
+
+Let's assume it is. Yes, our hypothetical test device has a touch screen,
+and we want to execute the test plan right away.
+
+First of all, make sure the information in the manifest file is correct:
+
+.. code-block:: none
+
+    $ cat /var/tmp/checkbox-ng/machine-manifest.json
+	
+    {
+      "com.canonical.certification::has_touchscreen": true
+    }
+
+Next, create a launcher file named ``auto-manifest`` with the following content:
+
+.. code-block:: none
+    :caption: auto-manifest
+    :name: auto-manifest
+    :emphasize-lines: 11-12
+
+    [launcher]
+    launcher_version = 1
+
+    [test plan]
+    unit = com.canonical.certification::tutorial-base-manifest
+    forced = yes
+    
+    [test selection]
+    forced = yes
+    
+    [ui]
+    type = silent
+
+You have previously learned about most of the sections in this launcher. One
+addition is the ``[ui]`` section, which covers customization related to
+the user interface. ``type = silent`` means Checkbox will run everything
+automatically. If there are any interactive jobs, they will be skipped. In
+addition, the System Manifest screen will be skipped and the values from the
+manifest file will be used. If the manifest file is absent, or if a required
+entry is absent, Checkbox will assume its value is ``False``.
+
+Now, run Checkbox using this launcher:
+
+.. code-block:: none
+
+    $ checkbox-cli launcher auto-manifest
```

Comment:
```suggestion
    $ checkbox-cli launcher <path>/auto-manifest
```
perhaps add a placeholder here to show the filepath?

Reply:
Fixed in [6bbfca4](https://github.com/canonical/checkbox/pull/736/commits/6bbfca44e8fb70498d8c1c78e25e01802740cdbb)